### DISCUSSION TITLE
我的Lantern突然用不了

### DISCUSSION THREAD

**Discussion by GloamZz** [getlantern/lantern-forum-cn#727] [Q&A]
我已经用了Lantern好几年了，也一直处于付费使用状态。上周还是能正常使用的，但自从前两天从关闭Lantern网页开始，之后就出了问题，至今无法连接完成翻墙，日志在附件中。

我的电脑配置：
版本：Windows 11 企业版
版本号：23H2
操作系统版本：22631.3880

网络供应商为电信，可正常浏览国内所有网络

以下是现象：
一、每次重启电脑后启动Lantern.exe，Lantern网站正常打开（Chrome浏览器），显示连接HK，但无法翻墙，刷新网站后网站报错如下
This site can’t be reachedlocalhost refused to connect.
Try:
Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED

二、启动Lantern.exe后，任务管理器出现Lantern进程，但没多久就自动关闭了，任务栏中也没有Lantern应用图标，所以它应该是没有正常启动

我尝试过的修复手段：
1、完全卸载迅雷
2、关闭防火墙
3、卸载Lantern，且移除AppData中的Lantern目录，然后在https://github.com/getlantern 上下载最新Window 10/7以上的不同版本应用程序，重新安装。
4、各种重置网络设置，重启电脑

[logs.zip](https://github.com/user-attachments/files/16285571/logs.zip)

**Comment by MaxMKMax** [comment]
@GloamZz 您好，谢谢您的反馈。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？需要用您受影响的版本和设备, 您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by GloamZz** [reply]
但是我现在无法启动lantern

![Uploading 1721321300302.png…]()

**Reply by GloamZz** [reply]
我双击Lantern.exe，网页加载失败，我没有办法访问Lantern应用程序

This site can’t be reachedlocalhost refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED

**Reply by GloamZz** [reply]
![Uploading 1721324227838.png…]()

刚才我下载使用App版本的Lantern，但是无法使用App的邮箱授权，输入邮箱后点击提交没有任何反应。

然后我点击go to Support然后select Report an Issue，选择了一个问题类型并输入描述，但是我点提交的时候它报错了。

错误
report_issue_error

**Reply by GloamZz** [reply]
并且我在Report页面没有办法修改email，它默认帮我填了一个我另一个gmail邮箱，它并不是我输入的，我都不知道他从哪里自动的获取了我的邮箱？

**Reply by GloamZz** [reply]
App的设置里面显示版本号1.0.0 构建版本1 SDK7.6.75

**Reply by MaxMKMax** [reply]
@GloamZz 
您好. 我帮您问问看怎么解决. 
https://github.com/getlantern/lantern
您这里试试看重新下载最新版本. 感觉您下载的不是最新版本.

**Comment by MaxMKMax** [comment]
@GloamZz 您好. 我帮您问问看怎么解决.
https://github.com/getlantern/lantern
您这里试试看重新下载最新版本. 感觉您下载的不是最新版本.

**Comment by MaxMKMax** [comment]
@GloamZz 可以帮我们用手动方法给我们log. 谢谢. 开会我帮你们反馈问题给技术团队, 尽快修复. 麻烦你啦. 谢谢帮忙.

非常抱歉影响了您的体验感. 如果有谁还没提交报告的可以帮我们个忙. 您那里能帮我提交报告然后给我 8 票号.吗? 如果应用闪退, 提交不了, 您可以手动帮我找log然后提供给我吗? 非常感谢! 有log我们就能马上修复了. 麻烦您帮忙了. 谢谢!
可以是看这么找:
C:\Users\username\AppData\Roaming\Lantern
